src/dd_send.c

```diff
@@ -45,8 +46,6 @@ DDReq *DDR_init(DDReq *req) {
     if (!req)
       return NULL;
     req->ownership = 0x01;
-  } else {
-    req->ownership = 0;
```

Comment:
Any reason this was removed? I'm guessing it accidentally got lost in rebase/merge or something.

Reply:
Oops, wrong brain parity.  Hopefully this is the last two-way merge we have to do, since I changed the way things are done on the other side.  Good catch!  Updated.